### DISCUSSION TITLE
Dreame L10s Pro and D10s Pro support

### DISCUSSION THREAD

**Discussion by Norbell** [Hypfer/Valetudo#1718] [Q&A / Support] [answered]
Hello, 

I'm not sure if this is the right place to ask such a question, but I was not able to find any good answere on the internet. 
Does valetudo support the 2022 modells of Dreame? 
Like the D10s Pro and the L10s Pro?

Are the firmware versions of the two models compatible with the older D10, Z10 and L10 devices? 

Thank you very much!

**Comment by Hypfer** [comment, marked as answer]
> Does valetudo support the 2022 modells of Dreame?

Technically yes, but there's no public root just yet.

> Are the firmware versions of the two models compatible with the older D10, Z10 and L10 devices?

No, those are different robots with their own firmwares.

See also: https://github.com/Hypfer/Valetudo/discussions/1690#discussioncomment-4710074
